src/main/java/edu/kis/powp/jobs2d/drivers/adapter/DriverAdapter.java

```diff
@@ -27,7 +28,10 @@ public void operateTo(int x, int y) {
 		line.setStartCoordinates(this.startX, this.startY);
 		line.setEndCoordinates(x, y);
 
-		drawLine(line);
+		startX = x;
+		startY = y;
+
+		DrawerFeature.getDrawerController().drawLine(line);
```

Comment:
Constructor injection should be used (using a parameter). This is unnecessary direct coupling with DrawerFeature class.